pyanaconda/modules/payloads/source/rpm_ostree/rpm_ostree.py

```diff
@@ -76,8 +77,7 @@ def network_required(self):
 
         :return: True or False
         """
-        # TODO: Implement this method
-        return False
+        return has_network_protocol(self.configuration.url)
```

Comment:
I'm not sure if this will work. I have a feeling that mirrors for rpm-ostree could be specified also as ```oci+https://...```, however, I wasn't able to find this anywhere.

Hi @dustymabe , could you please help us here and tell us. If rpm-ostree remote sources could be specified only by "common" prefixes (http, https, ftp...) or it supports also something special like the ``oci+http:///`` ? Thanks!

Reply:
I have changed the implementation, so it doesn't block this PR. See the commit https://github.com/rhinstaller/anaconda/pull/2871/commits/5a770dac2598b2c8d0a674d808f650442340820e.

Reply:
'oci+https://' is Flatpak-specific. (means "link to an index of a container registry") ostree only uses http/https.

Reply:
Thanks great for the reply. In that case I'm fine with this solution!